src/rust/mangastream/sources/genztoons/res/source.json

```
@@ -1,10 +1,10 @@
 {
 	"info": {
-		"id": "en.suryatoon",
+		"id": "en.genz",
 		"lang": "en",
-		"name": "Surya Toon",
-		"version": 2,
-		"url": "https://suryatoon.com/",
+		"name": "GenZ Toon",
+		"version": 1,
+		"url": "https://genztoons.com/",
```

base url shouldn't have a trailing slash -- didn't catch this for suryatoon I guess
```suggestion
		"url": "https://genztoons.com",
```